frontend/src/features/research/activities/list/ActivityListView.tsx

```diff
@@ -0,0 +1,127 @@
+import { TableSort } from 'components/Table/TableSort';
+import { useApiResearchFile } from 'hooks/pims-api/useApiResearchFile';
+import useIsMounted from 'hooks/useIsMounted';
+import { defaultActivityFilter, IActivityFilter } from 'interfaces/IActivityResults';
+import { orderBy } from 'lodash';
+import { Api_Activity } from 'models/api/Activity';
+import React, { useCallback } from 'react';
+
+import { ActivityFilterForm } from './ActivityFilter/ActivityFilterForm';
+import { ActivityResults } from './ActivityResults/ActivityResults';
+import { AddActivityForm } from './ActivityResults/AddActivityForm';
+import * as Styled from './styles';
+
+export interface IActivityListViewProps {
+  researchFileId?: number;
```

Comment:
should be a generic file id. Maybe we need another field like ```fileType``` simlar to notes and documents

Reply:
renamed to fileId